packages/igx-templates/igx-ts-legacy/projects/side-nav-auth/files/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve -o",
-    "build": "ng build --configuration production",
```

Comment:
I notice you're missing a `"watch"` configuration for the `side-nav-auth` projects.

Reply:
https://github.com/IgniteUI/igniteui-cli/pull/1181/commits/1d5ccece7c74dc4f8e9ea9c96b04ce6d64b0cdb5